pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -128,9 +128,12 @@ type CommandInitOption struct {
 	RestConfig                         *rest.Config
 	KarmadaAPIServerIP                 []net.IP
 	HostClusterDomain                  string
+	WaitComponentReadyTimeout          int
 }
 
 // Validate Check that there are enough flags to run the command.
+//
+//nolint:gocyclo
```

Comment:
This label will make the codes can't be checked by CI, which could bring some risks.

Reply:
If I don't add//nolint: gocyclo comments, lint gocyclo won't pass, which should be caused by adding an additional if